app/controllers/comments_controller.rb

```diff
@@ -17,10 +25,12 @@ def new
 
   # GET /comments/1/edit
   def edit
+    authorize @comment
   end
 
   # POST /comments or /comments.json
   def create
+    authorize @comment
     @comment = Comment.new(comment_params)
```

Comment:
is the order of execution correct or should `authorize @comment` run after something else ?